docs/relying-parties/how-tos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+
+As mentioned in the [Introduction](#introduction), SubPlat has read-only access to Play Store subscription information. We will store Google IAP subscribers' data in our system and update the information when we receive notifications from RTDN, broadcasting updates in real time to the relevant RPs.
+
+### Getting the current subscription status for a FxA user
+
+Assuming you are already integrated with FxA generally (see [Prerequisites](#prerequisites)), are receiving subscription updates for other types of subscriptions (e.g. for PayPal or Stripe), and you have completed the [mapping of Play Store to Stripe plans](#map-google-iap-to-stripe-plans), no further work is required to continue receiving these updates for Google IAP subscribers. You can also pull this information from FxA with the existing [`/account/profile`](/ecosystem-platform/api#tag/Account/operation/getAccountProfile) endpoint without any further changes.
+
+### Registering a Google IAP subscriber
+
+SubPlat needs to be able to associate a Google IAP subscription in Google's system to a specific FxA user. This is done by registering each Google IAP subscriber with the [`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname) endpoint.
+
+By design, we allow only one FxA user per Play Store [`productId`](https://support.google.com/googleplay/android-developer/answer/140504#zippy=%2Ccreate-a-new-subscription), which is how Google uniquely identifies the user's device when the subscription was purchased.
+
+This endpoint can be used to register a new Google IAP subscriber or to migrate an existing Google IAP subscriber. For the latter, see [(Optional) Migrate existing Google IAP subscribers to SubPlat](#optional-migrating-existing-google-iap-subscribers-to-subplat).
+
+### (Optional) Migrating existing Google IAP subscribers to SubPlat
+
+This section only applies to products that currently have a direct integration with Google who need to migrate existing subscribers to SubPlat.
+
+This step can be done by a developer. Before proceeding, complete the steps under [Configuration](#configuration) and [SubPlat API Calls](#subplat-api-calls). See also the [Testing your integration](#testing-your-integration) section.
+
+To migrate existing subscribers to SubPlat, use the existing registration endpoint ([`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname)) to register each existing user with SubPlat.
+
+## Testing your integration
+
+Currently, there is no sandbox or separate test environment. To create a test subscription, append `(Stage)` to the end of a subscription title in the Google Play Console to differentiate from a Production subscription. This approach will enable RPs to test multiple scenarios to understand what to expect.
```

Comment:
Hmm not sure I understand what I would need to do here. What's a subscription title in the Google Play console? Maybe a support link or screenshot would help?